server/server.py

```diff
@@ -63,7 +63,7 @@ def set_game_state(table, puzzle, key, value):
 
 # TODO replace more efficient way of doing this
 def initialize_game_state(table):
-  GAME_STATE[table] = {}
+  GAME_STATE[table] = {'table': table}
```

Comment:
this should already be on the client in the cookie, see app.jsx for how to extract it

Reply:
do i have to create another socket listener?